React/DevSupport/RCTDevMenu.m

```diff
@@ -439,33 +439,38 @@ - (void)setDefaultJSBundle {
 #if TARGET_OS_OSX // [TODO(macOS ISS#2323203)
 - (NSMenu *)menu
 {
-  NSMenu *menu = nil;
-  if (_bridge) {
-    NSString *desc = _bridge.bridgeDescription;
-    if (desc.length == 0) {
-      desc = NSStringFromClass([_bridge class]);
-    }
-    NSString *title = [NSString stringWithFormat:@"React Native: Development\n(%@)", desc];
+  RCTDevSettings *devSettings = _bridge.devSettings; // TODO(OSS Candidate ISS#2710739)
```

Comment:
nit: this comment is not necessary: this whole block of code is already enclosed in `#if TARGET_OS_OSX // [TODO(macOS ISS#2323203)` ... `#else // ]TODO(macOS ISS#2323203)`

Reply:
This comment refers to other issue than macOS fork comment, the line has been copied from `showOnShake`:
* https://github.com/microsoft/react-native-macos/blob/5639653093d5b3f02ef64ea9f7440910446e8239/React/DevSupport/RCTDevMenu.m#L184